_includes/footer.html

```diff
@@ -26,8 +26,8 @@
     <!--- Social Media Logos and Links. --->
     <div class="footer__social-media footer__block d-flex py-2">
       {% for app in site.data.socialMedia %}
-        <a href="{{ app.link }}" target="_blank" class="px-1">
```

Comment:
Small deal but I've been new lining on the attributes when there are 3 or more of them or they have over 100 column width. See line 47 of this same file for reference.